### ISSUE TITLE
incorporate style changes and some readability edits for multiple SCs

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#2030]
A PR was created a few months back, which was not associated with a specific issue, and so was not caught as being ready for Survey.
https://github.com/w3c/wcag/pull/1834

@alastc , given the time that has elapsed, some of these files may be in conflict. I have no issue helping resolve any such conflicts, or having such files removed from this PR for the time being. I'd like to get what we can incorporated now in the interest of expediency.

**Comment by alastc** [comment]
Closing as the PRs are merged.
